lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -1670,7 +1785,12 @@ namespace Js
 
             if (!JavascriptOperators::IsUndefinedObject(args.Values[3]) && JavascriptString::Is(args.Values[3]))
             {
+#ifdef INTL_WINGLOB
                 locale = JavascriptString::FromVar(args.Values[3])->GetSz();
+#else
+                langtag = JavascriptString::UnsafeFromVar(args[3])->GetSz();
```

Comment:
What's the difference between `args[3]` and `args.Values[3]`? I'd hope it's an operator overload that makes them the same but we should double check. Once understood, shouldn't we aim to be consistent here?

Reply:
There is, we discovered it a few weeks ago. I didn't want to change a whole bunch of them and make the diff noisy.